R/create_overview_status.R

```diff
@@ -1,10 +1,11 @@
-#' couple data of each individual tree in different years
+#' @title couple data of each individual tree in different years
 #'
-#' This function groups all given information on the life stages of an individual tree during different measures.
+#' @description
+#' This function creates a unique ID for each tree, that allows to group (f.e. by use of make_table_wide) all given information on the life stages of an individual tree during different measures.
```

Comment:
Suggestie: layoutmatig duidelijk maken dat je naar een andere functie verwijst (dit zal in de website een grijze achtergrond krijgen en een link vormen naar de info van die functie)

```suggestion
#' This function creates a unique ID for each tree, that allows to group (f.e. by use of `make_table_wide()`) all given information on the life stages of an individual tree during different measures.
```